### PR TITLE
[RFR] - Trad users forms

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hi,

in order to translate users (and group) forms, I added translator to formBuilder component.
Also, I found a typo on main application and so on I fixed it.

Finaly, this pull request is about User application.

I think we need to keep 3 commits as they are unrelated. 

edit: wait for user application to be done.

### CODE REVIEW

**Comment by ndufreche** [comment]
Thanks to your contribution @mickaelandrieu but user app still work in progress. And i think is a little too soon to translate this application entirely.

**Comment by mickaelandrieu** [comment]
@ndufreche it's ok, I'll rebase (and finish) this contribution when you will consider your application done.

**Comment by ndufreche** [comment]
Ok, thanks again @mickaelandrieu.

**Comment by mickaelandrieu** [comment]
I will rebase this one tomorrow :)

**Comment by fkroockmann** [comment]
@mickaelandrieu, you can rebase ?

**Comment by mickaelandrieu** [comment]
I will asap.

**Comment by fkroockmann** [comment]
Ok, thanks dude :)

**Comment by mickaelandrieu** [comment]
aw ... i'll do it by saturday sorry was so busy with BackBee PHP core cleaning :/

**Comment by ndufreche** [comment]
No problem @mickaelandrieu

**Comment by mickaelandrieu** [comment]
@ndufreche rebased, but let me time to test it before merge please.

**Comment by mickaelandrieu** [comment]
@ndufreche ok for me, after review I will rebase my commits :cat:

File: src/tb/init.js [hunk 1]
```diff
@@ -70,6 +70,11 @@ define(function () {
                                 RestDriver.setBaseUrl(Core.get('api_base_url'));
                                 DriverHandler.addDriver('rest', RestDriver);
 
+                                /* we need the filter to be able to use the login form before load
+                                 * the application configuration
+                                 */
+                                Renderer.addFilter('trans', function (key) { return key; });
```
Comment (ndufreche) [inline src/tb/init.js]: This happen to late, login form is called by component!session

**Comment by fkroockmann** [comment]
Use translator into a config of form instead of formbuilder's template

**Comment by mickaelandrieu** [comment]
Ok, this need some refactoring. Again in WIP ^^

**Comment by fkroockmann** [comment]
yep ^^

File: src/tb/apps/user/templates/user/list.item.twig [hunk 2]
```diff
@@ -22,7 +22,7 @@
                         <a href="#" class="btn btn-default btn-sm fa fa-angle-left btn-service" data-service="updateStatus">
                             <span class="sr-only">{{ 'previous'|trans }}</span>
                         </a>
-                        <span class="bb5-button-selector-status bb-toggle-status">{{ user.activated() }}</span>
+                        <span class="bb5-button-selector-status bb-toggle-status">{{ user.activated() === true ? 'activated'|trans : 'deactivated'|trans }}</span>
```
Comment (mickaelandrieu) [inline src/tb/apps/user/templates/user/list.item.twig]: ping @ndufreche is this a correct syntax for nunjucks ? I'm not yet sure.

Comment (ndufreche) [inline src/tb/apps/user/templates/user/list.item.twig]: I'm not sure but I think this not a valid syntax.
I suggest you, more this syntaxe =>
```jinja
{% user.activated() %}{{ 'activated'|trans }}{% else %}{ {'deactivated'|trans }}{% endif %}
```

Comment (mickaelandrieu) [inline src/tb/apps/user/templates/user/list.item.twig]: Ok, I update now.

Comment (ndufreche) [inline src/tb/apps/user/templates/user/list.item.twig]: But I have a question an user is :

activated or deactivated
active or inactive
enable or dsable

Comment (mickaelandrieu) [inline src/tb/apps/user/templates/user/list.item.twig]: hmmm ... ping @clementbonfils , what do you think ?

**Comment by mickaelandrieu** [comment]
so ... what is the status of this pull request ?

**Comment by ndufreche** [comment]
:+1:

**Comment by mickaelandrieu** [comment]
@ndufreche rebased and commits squashed => it's up to you now :-)

**Comment by ndufreche** [comment]
Yeah, nice job, thanks.

**Pull request merged**